doc/src/welcome-to-atomvm.md

```diff
@@ -42,9 +42,12 @@ We think you will agree that AtomVM provides a compelling environment not only f
 
 ## Design Philosophy
 
-AtomVM is designed from the start to run on small, cheap embedded devices, where system resources (memory, cpu, storage) are tightly constrained.  The smallest environment in which AtomVM runs has around 512k of addressable RAM, some of which is used by the underlying runtime (FreeRTOS), and some of which is used by the AtomVM virtual machine, itself, leaving even less RAM for your own applications.  Where there is a tradeoff between memory consumption and performance, minimizing memory consumption (and heap fragmentation) always wins.
+AtomVM is designed from the start to run on small, cheap embedded devices, where system resources (memory, cpu, storage) are tightly constrained.  The smallest environment in which AtomVM runs has around 128k of addressable RAM, some of which is used by the AtomVM virtual machine, itself, leaving even less RAM for your own applications.  Where there is a tradeoff between memory consumption and performance, minimizing memory consumption (and heap fragmentation) always wins.
 
-From the developer's point of view, AtomVM is designed to make use of the existing tool chain from the Erlang and Elixir ecosystems.  This includes the Erlang and Elixir compilers, which will compile Erlang and Elixir source code to BEAM bytecode.  Where possible, AtomVM makes use of existing tool chains to reduce the amount of unnecessary features in AtomVM, thus reducing complexity, as well as the amount of system resources in use by the runtime.  AtomVM is designed to be as small and lean as possible, providing as many resources to user applications, as possible.
+From the developer's point of view, AtomVM is designed to make use of the existing tool chains from the Erlang, Elixir and Gleam ecosystems.  This includes the Erlang, Elixir and Gleam compilers, which will compile Erlang, Elixir. and Gleam source code to BEAM bytecode.  Where possible, AtomVM makes use of existing tool chains to reduce the amount of unnecessary features in AtomVM, thus reducing complexity, as well as the amount of system resources in use by the runtime.  AtomVM is designed to be as small and lean as possible, providing as many resources to user applications, as possible.
+
+AtomVM exhibits some significant differences and limitations compared to BEAM. These are documented
+in a dedicated [page](./differences-with-beam.md).
```

Comment:
This looks great. This is one of the first questions experienced BEAM developers are likely go have.